packages/web-components/src/divider/divider.ts

```diff
@@ -51,4 +58,44 @@ export class Divider extends FASTElement {
    */
   @attr({ mode: 'boolean' })
   public inset?: boolean;
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+
+    this.elementInternals.role = this.role ?? DividerRole.separator;
+
+    if (this.role !== DividerRole.presentation) {
+      this.elementInternals.ariaOrientation = this.orientation ?? DividerOrientation.horizontal;
+    }
```

Comment:
Should this should be in the `constructor()`?

Reply:
In this case, we only want it set based on the role, which we I don't think we technically know for certain in the constructor.